include/cling/Interpreter/Interpreter.h

```diff
@@ -195,6 +195,19 @@ namespace cling {
     ///\brief Worker function, building block for interpreter's public
     /// interfaces.
     ///
+    ///\param [in] input - The input being code completed.
+    ///\param [in] CompilationOptions - The option set driving the compilation.
+    ///\param [out] T - The cling::Transaction of the compiled input.
+    ///
+    ///\returns Whether the operation was fully successful.
+    ///
+    CompilationResult CodeCompleteInternal(const std::string& input,
```

Comment:
Can you merge this with codeComplete? No need for an internal function here, this was just to avoid code duplication in the other cases